plugins/callback/foreman.py

```diff
@@ -223,6 +223,32 @@ def send_facts(self):
                 self._display.warning(u'Sending facts to Foreman at {url} failed for {host}: {err}'.format(
                     host=to_text(host), err=to_text(err), url=to_text(self.foreman_url)))
 
+    def _build_log(self, data):
```

Comment:
I think the method changed and it is now [build_log](https://github.com/theforeman/foreman-ansible-modules/blob/develop/plugins/callback/foreman.py#L95)